target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-ad7200-c2600.dtsi

```diff
@@ -249,11 +249,42 @@
 
 &pcie0 {
 	status = "okay";
+
+	bridge@0,0 {
+		reg = <0x00000000 0 0 0 0>;
+		#address-cells = <3>;
+		#size-cells = <2>;
+		ranges;
+
+		wifi@1,0 {
+			compatible = "pci168c,0040";
```

Comment:
Shouldn't we use `"qcom,ath10k"` here?

Reply:
@Ansuel Ping. qcom,ath10k instead?

Reply:
I never tested the ath10k compatible. I put this as it's an exact description of the wifi card in the device

Reply:
Let's use it for now then.